### PR TITLE
map update with some upstream sync content

### PR DESCRIPTION
**Pull request by SomeCabbage**
# Description
Added defibrillator cabinets into medbay and sec medbay.
Added fax machines for fax fanatics.
Added some paper bins for paper fanatics.
Added some durathread to HoP's room.
Replaced the armory Mosin. It's hammer time now.
Tagged the Brig Disposals
Made the EVA room button actually work.

# Changelog
:cl: SomeCabbage
- tweak: Parkstation map tweaks. Defibrilators, fax machines, paperbins, and breaching hammer. Some other minor changes as well.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml [hunk 1]
```diff
@@ -75,4 +75,66 @@
...
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
+    - Paper
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: woah

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml (hunk 1 above)
Comment (SomeCabbage) [reply Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: I dunno how bins work this is what the other ones were like

Comment (Pspritechologist) [reply Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: Are you fucking kidding me

Comment (Pspritechologist) [reply Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: Anyway consider using office paper instead

**Comment by Pspritechologist** [comment]
Will the pr bot mind that second colon?

**Comment by SomeCabbage** [comment]
I'll change it just to be safe

File: Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml (hunk 1 above)
Comment (SomeCabbage) [reply Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: oh is there any difference besides visual?

**Comment by DEATHB4DEFEAT** [comment]
> Will the pr bot mind that second colon?

probably not, it looks for (roughly) `- change: text...` so just an extra `:` should have no effect within `text...`

File: Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml (hunk 1 above)
Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: hey wait isn't there a `count` field? look at the plushie crate or something

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: nevermind, why is this a separate component? D:

Comment (SomeCabbage) [reply Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]: apparently bins are different

**Comment by SomeCabbage** [comment]
due to how the bin component works, each item is individually defined which is dumb, but besides that theres nothing else to be done

File: Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml [hunk 2]
```diff
@@ -75,4 +75,67 @@
     whitelist:
       tags:
       - Document
-      - Write
\ No newline at end of file
+      - Write
+
+- type: entity
+  parent: PaperBin
```
Comment (Pspritechologist) [inline Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml:81]:
```suggestion
  parent: PaperBin
  suffix: Office Paper
```

File: Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml [hunk 3]
```diff
@@ -75,4 +75,68 @@
...
+  suffix: Office Paper
```
Comment (Pspritechologist) [inline Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml:82]:
```suggestion
```

File: Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml [hunk 4]
```diff
@@ -75,4 +75,68 @@
...
+  id: PaperBin50
+  suffix: 50
```
Comment (Pspritechologist) [inline Resources/Prototypes/Entities/Structures/Storage/paper_bin.yml]:
```suggestion
  id: PaperBin50
  suffix: "50, Office Paper"
```

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**